nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -320,3 +320,5 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The option `services.prometheus.exporters.pihole.interval` does not exist anymore and has been removed.
 
 - `k3s` can now be configured with an EnvironmentFile for its systemd service, allowing secrets to be provided without ending up in the Nix Store.
+
+- The `hardware.video.hidpi.enable` was renamed to `fonts.optimizeForVeryHighDPI` to be consistent with what it actually does. Please see the documentation for the new option to decide if you want to keep it enabled.
```

Comment:
Please see the comment right under the next headline above this

Reply:
Fixed, also removed the previous change's documentation.